instalooter/pages.py

```diff
@@ -254,6 +254,14 @@ def _getparams(self, cursor):
             "after": cursor,
         }
 
+    @staticmethod
+    def get_profile_info(username, session):
+        try:
+            data = ProfileIterator._user_data(username=username, session=session)
+            return data['data']['user']
+        except:
+            return None
```

Comment:
is there a way to find what was the error? 
The way it is written here we will not know what happen in case of an error

Reply:
there is no meaning for the errors other than the flow changed and it does not work anymore (and we will get this info by returning none)